go/api/commands.go

```diff
@@ -0,0 +1,19 @@
+// Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+// GenericCommands defines an interface to handle generic command and transaction requests for a standalone Redis client.
+type GenericCommands interface {
+	// CustomCommand executes a single command, specified by args, without checking inputs. Every part of the command,
+	// including the command name and subcommands, should be added as a separate value in args. The returning value depends on
+	// the executed command.
+	//
+	// This function should only be used for single-response commands. Commands that don't return response (such as SUBSCRIBE),
+	// or that return potentially more than a single response (such as XREAD), or that change the client's behavior (such as
+	// entering pub/sub mode on RESP2 connections) shouldn't be called using this function.
+	//
+	// For example, to return a list of all pub/sub clients:
+	//
+	//	client.CustomCommand([]string{"CLIENT", "LIST","TYPE", "PUBSUB"})
```

Comment:
can we add a response to this example?  what do we do with the returned payload/error?

Reply:
The other clients use this example command and do not do anything with the response. Let me know if you want me to add one here anyways though. It will be slightly verbose because I have to show error handling too. What should I do with the response? Just print it?
```go
result, err := client.CustomCommand([]string{"CLIENT", "LIST","TYPE", "PUBSUB"})
if err != nil {
    log.Fatal("error executing command: ", err)
}

fmt.Printf("result: %s", result)
```